forum/users/models.py

```diff
@@ -1,3 +1,57 @@
 from django.db import models
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin,
+)
+import datetime
+import re
+from phonenumber_field.modelfields import PhoneNumberField
 
-# Create your models here.
+ROLE_CHOICES = [
+    (0, "Unassigned"),
+    (1, "Startup"),
+    (2, "Investor"),
+]
```

Comment:
At the team meeting on December 16, it was decided to add the field `Both` to `ROLE_CHOICES`:  

```python
ROLE_CHOICES = [
    (0, "Unassigned"),
    (1, "Startup"),
    (2, "Investor"),
    (3, "Both")
]
```